packages/fungible-token/bridge-fungible-token/implementation/src/main.sw

```diff
@@ -62,7 +61,7 @@ use utils::{
     encode_data,
     encode_register_calldata,
 };
-use src_20::SRC20;
+use standards::src20::SRC20;
```

Comment:
Was this change not done in https://github.com/FuelLabs/fuel-bridge/pull/201?

Reply:
Yeah I am not sure what is exactly flagging this as a change. Probably because the file has moved and because of the multiple rebases / merges that have happened since this PR started, git is not able to exactly track what happened here.